### ISSUE TITLE
Cropland: Refugee LCLUC AOIs February (2015-2024)-(Annual or quinquennial)

### ISSUE THREAD

**Issue by cnakalembe** [nasaharvest/crop-mask#337]
- [ ] Labeling projects created and links added below
- [ ] [Set 1]() Labeling
- [ ] [Set 2]() Labeling
- [ ] Data added to repository
- [ ] Model trained
- [ ] Map made
- [ ] Expert check

**Comment by hannah-rae** [comment]
@MsPixels will confirm details with @cnakalembe and break out into individual issues

**Comment by cnakalembe** [comment]
We have to do annual-- something worth exploring later which is outside the current scope is looking at rapid changes (month to month) when new settlements are established

**Comment by MsPixels** [comment]
New CEO Institution created for this project @cnakalembe

**Comment by MsPixels** [comment]
CEO Labeling update: 53.6% analyzed for Set 1 and 50.70% completed for Set 2

**Comment by MsPixels** [comment]
@hannah-rae, CEO Labeling Update: 902 (90.20%) for Set A and 673 (67.30%) for Set B

**Comment by MsPixels** [comment]
Intercomparison analysis for Uganda North @cnakalembe 

![Image](https://github.com/nasaharvest/crop-mask/assets/110104000/72e90cde-a7db-4a02-97d7-da158514fc66)

**Comment by MsPixels** [comment]
After adding corrective labels, here is the metrics @cnakalembe:

"Uganda_North_2022_V2": {
        "params": "https://wandb.ai/nasa-harvest/crop-mask/runs/0ngyleht",
        "test_metrics": {
            "accuracy": 0.6388,
            "f1_score": 0.5341,
            "precision_score": 0.3917,
            "recall_score": 0.8393,
            "roc_auc_score": 0.8004
        },
        "val_metrics": {
            "accuracy": 0.6886,
            "f1_score": 0.6321,
            "precision_score": 0.488,
            "recall_score": 0.8971,
            "roc_auc_score": 0.8259
        }
    },

**Comment by MsPixels** [comment]
Corrective Labeling [App](https://ee-aasareansah.projects.earthengine.app/view/corrective-labeling-app---northern-uganda2022)

**Comment by MsPixels** [comment]
Generated a Random Forest approach in GEE to train the machine learning model and compare the results with the LSTM result. 2975 crop and non crop labels were collected as training data and the CEO labeled dataset was used as the validation set. [Code](https://code.earthengine.google.com/5ec6dc6b9da264070045f188c7b1cf69)

**Comment by MsPixels** [comment]
The 2975 labels were imported into the LSTM data pipeline and used to retrain the model. The metrics of the model is stated below:

Uganda_North_2022_V3": {
        "params": "https://wandb.ai/nasa-harvest/crop-mask/runs/9plwjt8m",
        "test_metrics": {
            "accuracy": 0.8106,
            "f1_score": 0.6446,
            "precision_score": 0.6,
            "recall_score": 0.6964,
            "roc_auc_score": 0.8742
        },
        "val_metrics": {
            "accuracy": 0.8465,
            "f1_score": 0.7445,
            "precision_score": 0.7391,
            "recall_score": 0.75,
            "roc_auc_score": 0.9045
        }

**Comment by MsPixels** [comment]
<img width="620" alt="Screenshot 2024-06-13 at 1 36 28 PM" src="https://github.com/nasaharvest/crop-mask/assets/110104000/41ec96fc-10c6-435e-92fd-5f38401c17cf">

[Link](https://docs.google.com/spreadsheets/d/1lBhE3dvci5AzYxQHO_7I31LfHO42uaLZiLtX4cXef58/edit?gid=0#gid=0)

**Comment by MsPixels** [comment]
[LSTM vrs RF 2022](https://code.earthengine.google.com/027bf064f99ab48768d29632aa36ba37)
